pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -66,8 +60,8 @@ stdenvNoCC.mkDerivation rec {
   outputHashAlgo = "sha256";
   outputHashMode = "recursive";
   outputHash = if stdenvNoCC.isDarwin
-    then "1224dln7n8px1rk8biiggf77wjhxh8mzw0hd8zlyjm8i6j8w7i12"
-    else "0d8ai0wi8drp071acqkm1wv6vyg12010y843y56zzi1pql81xqvx";
+    then "0000000000000000000000000000000000000000000000000000"
```

Comment:
@smaret can you run `nix-build -A mkl` and let me know what the Darwin hash is, as you did last time?
  https://github.com/NixOS/nixpkgs/pull/47182#discussion_r222521464

BTW, is there a way for me to cross compile to Darwin and compute the hash on my own, or is that only available for FOSS platforms?